### PR TITLE
Show config errors in the VS Code extension

### LINKED ISSUE #392: VS Code extension does not tell you when your standard library/config is wrong

### PR DESCRIPTION
**Pull request by Kampfkarren**
Closes #392

### CODE REVIEW

File: selene/src/validate_config.rs [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+impl From<serde_yaml::Location> for ErrorRange {
+    fn from(location: serde_yaml::Location) -> Self {
+        Self {
+            start: location.index(),
+            end: location.index(),
+        }
+    }
+}
+
+// TODO: Test
```
Comment (matthargett) [inline selene/src/validate_config.rs:48]: Why not add the test in the same PR?

**Review by matthargett — CHANGES_REQUESTED** [review]
Lots of TS file changes, but no tests covering the changes. You could do a pre-release without tests to elicit focused feedback, but without tests this seems like a lot to push to users.

File: selene/src/validate_config.rs (hunk 1 above)
Comment (Kampfkarren) [reply selene/src/validate_config.rs:48]: All the TODOs will be solved before I undraft the PR, but I knew I'd forget if I didn't write it down

**Comment by Kampfkarren** [comment]
I think the VS Code extensions have been ridiculously hard to even make simple tests for, I'll see what I can do

**Comment by Kampfkarren** [comment]
Going to ship a pre-release version of the VSC extension when this is ready

**Comment by Kampfkarren** [comment]
Forgot changelog, will commit that later

**Pull request merged**